apps/studio/src/components/AppleLogin.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Logs the user into the app using the Apple Auth API.
+ */
+
+import { FunctionComponent, ReactNode } from "react";
+import {
+  AppleAuthResponse,
+  appleAuthHelpers,
+  useScript,
+} from "react-apple-signin-auth";
+import AppleIcon from "@mui/icons-material/Apple";
+import { Button } from "@newm-web/elements";
+import { VITE_APPLE_CLIENT_ID } from "@newm-web/env";
+import { setToastMessage } from "../modules/ui";
+import { useAppleLoginThunk } from "../modules/session";
+import { useAppDispatch } from "../common";
+
+interface Props {
+  readonly children?: ReactNode;
+}
+
+const AppleLogin: FunctionComponent<Props> = ({ children }) => {
+  useScript(appleAuthHelpers.APPLE_SCRIPT_SRC);
+  const dispatch = useAppDispatch();
+  const [logIn] = useAppleLoginThunk();
+
+  const redirectUri = `${window.location.origin}/apple`;
```

Comment:
I think this should be `window.location.href` instead of apple so it matches the appropriate page where the button/function is being called? We have the two sign up paths already enabled through the Apple dev backend as well. This format was used in the previous linkedIn file but I think we were using a `LinkedInCallback` route with a `/linkedin` path.

Reply:
Updated to `window.location.href` to match the location of the NEWM login and sign-up pages where the apple sign will be located.